src/main/resources/com/owlcyberdefense/ntp/xsd/baseFormat.dfdl.xsd

```diff
@@ -18,9 +18,10 @@
                    byteOrder="bigEndian"
                    bitOrder="mostSignificantBitFirst"
                    encoding="ASCII"
-                   lengthUnits='bits'
+				   lengthKind='explicit'
```

Comment:
Really should be implicit. But let's not fix this now. Let's park a ticket for it.